spec/jobs/plexer_job_spec.rb

```diff
@@ -52,7 +52,7 @@
     end
 
     it 'adds ZipPart to each related ZMV' do
-      expect(cm.zipped_moab_versions.count).to eq 2
+      expect(cm.zipped_moab_versions.count).to eq 3
       job.perform(druid, version, s3_key, metadata)
       expect(parts1.map(&:md5)).to eq [md5]
       expect(parts2.map(&:md5)).to eq [md5]
```

Comment:
i think we should probably have also created a `parts3`, and added an expectation about it similar to `parts1` and `parts2`.

also, i can't comment on it directly, because the line isn't part of the diff, but i'd also be in favor of adding a third expectation about the IBM delivery job to the `it 'splits the message out to endpoints' do`, between what are currently lines 46 and 47:  https://github.com/sul-dlss/preservation_catalog/pull/1136/files#diff-419c6db3143382d622d1582fd8e253fbR47

Reply:
oh and then along with that, we should update this comment to be something like `# new 3rd endpoint ensures cm has 3 ZMVs`: https://github.com/sul-dlss/preservation_catalog/pull/1136/files#diff-419c6db3143382d622d1582fd8e253fbR35